pillar.example

```diff
@@ -122,3 +122,9 @@ packages:
   remote_pkgs:
     zoom: 'https://zoom.us/client/latest/zoom_amd64.deb'
 
+  retry_option:
```

Comment:
You missed a spot here :yum:

Reply:
Thanks @javierbertoli for reviewing. all fixed now.